api/server/routes_create_update.go

```diff
@@ -66,10 +89,16 @@ func (s *Server) changeRoute(ctx context.Context, wroute *models.RouteWrapper) e
 	return nil
 }
 
-// ensureApp will only execute if it is on put
-func (s *Server) ensureRoute(ctx context.Context, method string, wroute *models.RouteWrapper) (routeResponse, error) {
+// ensureApp will only execute if it is everything except PATCH
```

Comment:
this is above ensureRoute now, so this is no longer true [of ensureRoute]

Reply:
fixed